### PR TITLE
fix: scatter bubble null data

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by shiren — APPROVED** [review]
LGTM

File: apps/chart/src/component/bubbleSeries.ts [hunk 1]
```diff
@@ -110,29 +118,31 @@ export default class BubbleSeries extends Component {
...
-          type: 'circle',
-          radius,
-          color,
-          style: ['default'],
-          seriesIndex,
-          name,
-          borderWidth,
-          borderColor,
-        });
+        if (!isNull(datum)) {
```
Comment (lja1018) [inline apps/chart/src/component/bubbleSeries.ts:121]: `filter`로 한번 걸고 if 문 없애는건 어떤가요?

File: apps/chart/tests/components/bubbleSeries.spec.ts [hunk 2]
```diff
@@ -5,236 +5,358 @@ import BubbleSeries from '@src/component/bubbleSeries';
...
+    });
   });
-});
 
-it('should register closest responder to the mouse', () => {
-  const closestResponder = result.responders[0];
-  const distantResponder = result.responders[1];
+  it('should register closest responder to the mouse', () => {
+    const closestResponder = result.responders[0];
+    const distantResponder = result.responders[1];
```
Comment (lja1018) [inline apps/chart/tests/components/bubbleSeries.spec.ts:215]: `const [closestResponder, distantResponder] = result.responders;`는 어떤가요?

**Review by lja1018 — APPROVED** [review]
리뷰완료
고생하셨습니다!

File: apps/chart/tests/components/bubbleSeries.spec.ts [hunk 3]
```diff
@@ -5,236 +5,358 @@ import BubbleSeries from '@src/component/bubbleSeries';
...
+        type: 'circle',
+        x: 280,
+        y: 280,
+        name: 'nameB',
+      },
+    ],
+  };
 
-  expect(bubbleSeries.models.series[2].color).toEqual('rgba(187, 187, 187, 0.1)');
+  it(`should make models properly when calling render`, () => {
```
Comment (jungeun-cho) [inline apps/chart/tests/components/bubbleSeries.spec.ts:352]: 여긴 백틱 안써도될것같아요~

File: apps/chart/tests/components/scatterSeries.spec.ts [hunk 4]
```diff
@@ -202,40 +345,16 @@ const result = {
...
-  it(`should make ${modelName} properly when calling render`, () => {
-    expect(scatterSeries[modelName]).toEqual(result[modelName]);
-  });
-});
+  };
 
-it('should register closest responder to the mouse', () => {
-  const closestResponder = result.responders[0];
-  const distantResponder = result.responders[1];
+  it(`should make models properly when calling render`, () => {
```
Comment (jungeun-cho) [inline apps/chart/tests/components/scatterSeries.spec.ts:350]: 여기도요~!

**Review by jungeun-cho — APPROVED** [review]
확인했습니다.~ 수고하셨어요.

**Pull request merged**